source/_components/device_tracker.icloud.markdown

```diff
@@ -48,7 +50,7 @@ To disable the drainage of the battery, a dynamic interval is being used for eac
 2 Factor Authentication is the improved version of 2 Steps Authentication, this is still not supported by the pyicloud library. Therefore it's not possible to use it with the device_tracker yet.
 
 4 services are available for this component:
-- **icloud_update**: This service can be used to ask for an update of a certain iDevice. The `account_name` and `device_name` are optional.
+- **icloud_update**: This service can be used to ask for an update of a certain iDevice. The `account_name` and `device_name` are optional. Request will result in new Home Assistant event describing current iphone location. Can be used in automations when manual location update is needed, e.g. to check if anyone is home when door's been opened.
```

Comment:
Can you indicate what event is fired?

Reply:
Updated with link to state_changed event.